src/main/java/seedu/weme/commons/util/FileUtil.java

```diff
@@ -37,12 +132,26 @@ public static boolean isValidPath(String path) {
         return true;
     }
 
+    /**
+     * Returns true if {@code path} can be converted into a {@code Path} via {@link Paths#get(String)}
+     * and {@link Files#exists(Path)}, otherwise returns false.
+     * @param path A string representing the file path. Cannot be null.
+     */
+    public static boolean isValidDirectoryPath(String path) {
+        try {
+            Paths.get(path);
```

Comment:
```suggestion
            return Files.isDirectory(Paths.get(path));
```

Reply:
in fact if you have a path object, you can just directly use Files.isDirectory(path)